### DISCUSSION TITLE
Compiling on Linux

### DISCUSSION THREAD

**Discussion by assemblaj** [ikemen-engine/Ikemen-GO#1675] [Other] [answered]
I figure since Linux may have particular build needs, a discussion thread on compiling for Linux might be useful. 

I'm using PopOS. I recently compiled on Linux for the first time, it required 'gtk-3' and 'alsa' for a few of the Go packages. I've updated the wiki accordingly. 

I feel like some of the packages listed in the install command for Linux might not be used anymore (feel like Sam may have gotten rid of Open AL, for example), but I left them be because I'm not sure.

**Comment by K4thos** [comment, marked as answer]
Open AL is indeed not used anymore, should be deleted from wiki
